server/gitrest/packages/gitrest-base/src/utils/redisFs/redisFsManager.ts

```diff
@@ -239,7 +239,7 @@ export class RedisFs implements IFileSystemPromises {
 		const folderpathString = folderpath.toString();
 
 		await executeRedisFsApi(
-			async () => this.redisFsClient.set(folderpathString, RedisFSConstants.Folder),
+			async () => this.redisFsClient.set(folderpathString, ""),
```

Comment:
Having a not empty string breaks the getcommits flow..specifically the call to git.log fails